src/venue/service.js

```diff
@@ -146,6 +146,27 @@ angular
       };
 
       /*==============================================================*
+      /* Info
+      /*==============================================================*/
+
+
+      VenueService.prototype.listInfo = function listInfo(venueId) {
```

Comment:
Definitely not necessary by any means, but I've learned to just add support for it rather than write the function and then one day later have to go update the sdk anyway because now I need to pass more args.